src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -1154,6 +1156,40 @@ private static void ReadMSBuildMetadata(JsonTextReader jsonReader, PackageSpec p
 
                         warningProperties = new WarningProperties(warnAsError, noWarn, allWarningsAsErrors, warningsNotAsErrors);
                         break;
+                    case "SdkAnalysisLevel":
+                        string skdAnalysisLevelString = jsonReader.ReadNextTokenAsString();
+
+                        if (!string.IsNullOrEmpty(skdAnalysisLevelString))
+                        {
+                            try
+                            {
+                                sdkAnalysisLevel = new NuGetVersion(skdAnalysisLevelString);
+                            }
+                            catch (ArgumentException ex)
+                            {
+                                throw new ArgumentException(string.Format(CultureInfo.CurrentCulture, Strings.Invalid_AttributeValue, "SdkAnalysisLevel", skdAnalysisLevelString, "9.0.100"), ex);
+                            }
+                        }
+                        break;
+
+                    case "UsingMicrosoftNETSdk":
+                        string usingMicrosoftNetSdkString = jsonReader.ReadNextTokenAsString();
```

Comment:
Newtonsoft.Json has a `ReadAsBool()` API: https://www.newtonsoft.com/json/help/html/M_Newtonsoft_Json_JsonReader_ReadAsBoolean.htm

Can it be used?